File: HELLO_JAVA/src/day03/Animal.java
package day03;

public class Animal {
	int full = 1;
	
	void eat(int amount) {
		full += amount;
	}
	
	public static void main(String[] args) {
		Animal ani = new Animal();
		System.out.println("ani.full : "+ani.full);
		ani.eat(9);
		System.out.println("ani.full : "+ani.full);
	}
	
}

File: HELLO_NODE/animal.js
class Animal {
	constructor() {
		this.full = 1;
	}
	
	eat(amount) {
		this.full += amount;
	}
}

/*var ani = new Animal();
console.log(ani.full);
ani.eat(9);
console.log(ani.full);*/

module.exports = Animal;

if (require.main === module) {
	var ani = new Animal();
	console.log("1", ani.full);
	ani.eat(9);
	console.log("1", ani.full);
}
